### PR TITLE
feat: change some telemetry metrics

### PR DESCRIPTION
**Pull request by thalesmg**
- Originally, we wanted to just check if certain features were enabled.  Now, we want to count certain usage metrics for them.
- Bridges being reference in `from` clauses in rule actions were not being accounted for.

### CODE REVIEW

File: scripts/format_app.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 def handle_cli_args():
   parser = argparse.ArgumentParser(description='A script for formatting a given app within emqx')
-  parser.add_argument("-a", "--application", required=True, help="The relative path to the application which is to be formatted.")
+  parser.add_argument("-a", "--application", required=True, action="append", help="The relative path to the application which is to be formatted.")
```
Comment (mononym) [inline scripts/format_app.py:14]: Should this be pulled out into its own commit since it has nothing to do with telemetry? I don't know why it would, but if for some reason the telemetry commit had to be rolled back you'd lose this too.

Comment (thalesmg) [reply scripts/format_app.py:14]: Yes, there is a commit with only this change: 15dcf8aaa3db60a408cc67d1298f6c7f6f6ce1f0

**Review by qzhuyan — APPROVED** [review]
